torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -253,17 +253,21 @@ void CudaPrinter::visit(const LetStmt* v) {
 }
 
 void CudaPrinter::visit(const IfThenElse* v) {
-  os() << "(";
+  os() << "((";
   v->condition()->accept(this);
   os() << ") ? ";
   v->true_value()->accept(this);
   os() << " : ";
   v->false_value()->accept(this);
+  os() << ")";
 }
 
 class PrioritizeLoad : public IRMutator {
  public:
-  virtual const Expr* mutate(const Load* v) {
+  const Expr* mutate(const Load* v) override {
+    if (nested_if_then_else_ > 0) {
```

Comment:
You might add a comment here that points ahead to the bigger comment at the variable declaration.

Reply:
Done